protostar/commands/build_command.py

```diff
@@ -1,15 +1,27 @@
 from pathlib import Path
 from typing import List, Optional, Any
+from dataclasses import dataclass
 
 from protostar.cli import ProtostarArgument, ProtostarCommand, MessengerFactory
 from protostar.cli.common_arguments import COMPILED_CONTRACTS_DIR_ARG
 from protostar.compiler import ProjectCompiler, ProjectCompilerConfig
-from protostar.io import LogColorProvider, Message
+from protostar.starknet_gateway import SuccessfulBuildResponse
+from protostar.io import StructuredMessage, LogColorProvider
 
 
-class BuildActivityMessageTemplate(Message):
+@dataclass
+class SuccessfulBuildMessage(StructuredMessage):
+    response: SuccessfulBuildResponse
+
     def format_human(self, fmt: LogColorProvider) -> str:
-        return "Building projects' contracts"
+        lines: list[str] = ["Building projects' contracts"]
+        for contract_name, class_hash in self.response.class_hashes.items():
+            lines.append(f"Class hash for contract {contract_name}: {class_hash}")
+
+        return "\n".join(lines)
+
+    def format_dict(self) -> dict:
+        return self.response.class_hashes
```

Comment:
Class hashes should be represented as hexes preceded with `0x`.

Reply:
[convert class hashes to the hex format](https://github.com/software-mansion/protostar/pull/1244/commits/98751f8e3808ac887f999c58b31a3a5918f00bc0)